tutorials/integrations/local-subsquid.md

```diff
@@ -567,9 +408,17 @@ processor.run(new TypeormDatabase(), async (ctx) => {
 
 在终端中，您应当能看见检索器在处理区块！
 
-![Spin up a Subsquid indexer](/images/tutorials/integrations/local-subsquid/local-squid-7.webp)
+![Run sqd process](/images/tutorials/integrations/local-subsquid/local-squid-6.png)
+
+如果您的Squid没有正确地检索区块，请确保您的开发节点正在使用`--sealing`标志运行。以本教程例子来说，你应该将标志设置为`--sealing 4000`，这样每四秒就会产生一个区块。您也可以根据需要随意编辑时间间隔。在您尝试再次启动您的Squid之前，请运行以下指令来重新启动Squid：
 
-如果您的Squid没有正确地检索区块，请确保您的开发节点正在使用`--sealing`标志运行。以本教程例子来说，你应该将标志设置为`--sealing 4000`，这样每四秒就会产生一个区块。您也可以根据需要随意编辑时间间隔。在您尝试再次启动您的Squid之前，请运行以下指令来关闭本地Archive和Squid：
+1. 关闭您的Squid：
+
+    ```bash
+    sqd down
+    ```
+
+2. 重新启动您的Squid：
```

Comment:
This is the closest I could get to the actual line that needs to be updated. Starting at line 423. But this set of steps should have a number 3, and the sqd archive-up && sqd up commands can be removed. It should look like the following:

```
1. Shut down your Squid

    ```bash
    sqd down
    ```

2. Start your Squid back up:

    ```bash
    sqd up
    ```

3. Start indexing again:

    ```bash
    sqd process
    ```
```